src/main/anovos/data_analyzer/association_evaluator.py

```diff
@@ -146,17 +147,15 @@ def variable_clustering(
     idf,
     list_of_cols="all",
     drop_cols=[],
-    sample_size=100000,
     stats_mode={},
     print_impact=False,
 ):
     """
     Variable Clustering groups attributes that are as correlated as possible among themselves within a cluster and
```

Comment:
We can say in here that variable_clustering is a wrapper for our VarClusHiSpark, and briefly introduce the function and algorithm more.